SpaceShooter/SpaceShooter/Box.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Console;
+
+namespace Space_Shooter
+{
+    internal class Box
+    {
+        private (int x, int y) Position;
+        private int Width;
+        private int Height;
+        private string Title;
+
+        private (int x, int y) CurrentPos;
+
+        public Box(int width, int height, (int x, int y) position, string title)
+        {
+            Position = position;
+            Width = width;
+            Height = height;
+            Title = title;
+        }
+        public void InitBox(bool title_bar, GridBuffer buffer)
```

Comment:
```suggestion
        public void InitBox(bool titleBar, GridBuffer buffer)
```

Camel-case for parameters